src/Raven.Server/Utils/BackupUtils.cs

```diff
@@ -657,6 +657,14 @@ await BackupHelper.RunWithRetriesAsync(maxRetries: 10, async () =>
 
             backupResult?.AddError($"{message}{Environment.NewLine}{fullException}");
         }
+        finally
+        {
+            var database = serverStore.DatabasesLandlord.TryGetOrCreateResourceStore(databaseName).Result;
+            var retentionPeriod = database.Configuration.Backup.BackupHistoryRetentionPeriod;
+
+            serverStore.DatabaseInfoCache.BackupHistoryStorage.StoreBackupStatus(databaseName, status, retentionPeriod);
```

Comment:
- Can `StoreBackupStatus` and `StoreBackupResultDetails` be in the same command so we won't have to open the write transaction twice?
- Need some retry policy in case we fail to save it.